C++/C++.sublime-syntax

```diff
@@ -694,6 +699,18 @@ contexts:
       scope: punctuation.accessor.arrow.c++
       push: members-after-accessor-junction
 
+  
+
+  using-alias:
+    # consume keyword if followed by typename
+    - match: '\b(using)\b(?=\s+typename\b)'
+      captures:
+        1: keyword.control.c++
+    - match: '\b(using)\b\s+({{identifier}})(?!\s*(<|::))'
+      captures:
+        1: storage.type.c++
```

Comment:
This should consistently be `keyword.control.c++`. I don't think the scope should change to `storage.type`.

Reply:
The reasoning behind this was that `typedef`, `class`, `enum`, etc. all have the keyword `storage.type` and I wanted it to be consistent. On the other hand, `namespace` when used for namespace aliases – as in `namespace foo = bar;` – doesn't change its scope either and sticks with `keyword.control`. For me both choices seem reasonable so I will leave it up to you.